Formula/needle.rb

```diff
@@ -1,9 +1,10 @@
 class Needle < Formula
   desc "Compile-time safe Swift dependency injection framework with real code"
   homepage "https://github.com/uber/needle"
+  # TODO: Check if a GitHub tarball is sufficient here.
   url "https://github.com/uber/needle.git",
```

Comment:
We can probably use a GitHub tarball here, but I want to test my build fix first.